app/models/reward.rb

```diff
@@ -0,0 +1,5 @@
+class Reward < ApplicationRecord
+  validates :title, :description, presence: true, allow_nil: false
```

Comment:
with `presence: true` nil and an empty string fails validation, so no need to add `allow_nil: false` 
```suggestion
  validates :title, :description, presence: true
```